cypress/fixtures/single_recipe.json

```diff
@@ -0,0 +1,11 @@
+{ 
+  "recipes":
+  [
```

Comment:
It is also NOT an array. You need to check the request specs on the back end before creating fixtures.

Reply:
Our bad, lack of brain strength, resolved with upcoming push.